src/components/lib/Tooltip.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useRef, useState } from "react"
+import classNames from "classnames"
+import PropTypes from "prop-types"
+
+const Tooltip = ({
+  triggerElement = <button>Help</button>,
```

Comment:
Hmmm… specifying the trigger element here rather than as a child, but then inserting it as a child, is a bit counterintuitive. I don't have a strong alternative suggestion in my head at the moment, but will ponder.

Reply:
I wasn't sure how to go about padding the trigger element as the child and attaching the listener to toggle the tooltip closed. I considered making the only the element type configurable and pass in a label (two separate props), but then I thought about the case when maybe you wanted a whole div with other child content or something (not just a button) to trigger a tooltip.

Reply:
Haha yeah I went through the same thought process. *Feels* like there has to be a thing :thinking: